### ISSUE TITLE
Question about tf.distribute.strategy

### ISSUE THREAD

**Issue by rohanmuplara** [tensorflow/tensorflow#48912]
Thank you for submitting a TensorFlow documentation issue. Per our GitHub
policy, we only address code/doc bugs, performance issues, feature requests, and
build/installation issues on GitHub.

The TensorFlow docs are open source! To get involved, read the documentation
contributor guide: https://www.tensorflow.org/community/contribute/docs

## URL(s) with the issue:
https://www.tensorflow.org/guide/keras/writing_a_training_loop_from_scratch

## Description of issue (what needs changing):
Confused on how the losses get added up

### Clear description

I had one additional question for this line in documentation  (training_loss.update_state(loss * strategy.num_replicas_in_sync)
Why do we multiply the loss by number of replicas. I understand the gradients get added up but what happens to the loss. Does that get added up?
If not, s this randomly choosing the loss on one replica and multiplying by the number of replicas?

```
@tf.function
def train_multiple_steps(iterator, steps):
  """The step function for one training step"""

  def step_fn(inputs):
    """The computation to run on each TPU device."""
    images, labels = inputs
    with tf.GradientTape() as tape:
      logits = model(images, training=True)
      loss = tf.keras.losses.sparse_categorical_crossentropy(
          labels, logits, from_logits=True)
      loss = tf.nn.compute_average_loss(loss, global_batch_size=batch_size)
    grads = tape.gradient(loss, model.trainable_variables)
    optimizer.apply_gradients(list(zip(grads, model.trainable_variables)))
    training_loss.update_state(loss * strategy.num_replicas_in_sync)
    training_accuracy.update_state(labels, logits)

  for _ in tf.range(steps):
```

### Correct links

Is the link to the source code correct?
yes

### Parameters defined

Are all parameters defined and formatted correctly?
yes

### Returns defined

Are return values defined?
yes

### Raises listed and defined

Are the errors defined? For example,
https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/feature_column/categorical_column_with_vocabulary_file#raises

Yes
### Usage example

Is there a usage example?

Yes

### Request visuals, if applicable
Not applicable

### Submit a pull request?

Are you planning to also submit a pull request to fix the issue? See the docs
contributor guide: https://www.tensorflow.org/community/contribute/docs,
docs API guide: https://www.tensorflow.org/community/contribute/docs_ref and the
docs style guide: https://www.tensorflow.org/community/contribute/docs_style

**Comment by 8bitmp3** [comment]
cc @MarkDaoust @lamberta (a docs-related question about https://www.tensorflow.org/guide/keras/writing_a_training_loop_from_scratch)

**Comment by rmothukuru** [comment]
@rohanmuplara,
Can you please give reference to the **`Tensorflow Documentation`** where the code you specified, 
`training_loss.update_state(loss * strategy.num_replicas_in_sync ` is mentioned? Thanks!

**Comment by 8bitmp3** [comment]
@rohanmuplara @rmothukuru The code snippet may be from the [Use TPUs](https://www.tensorflow.org/guide/tpu#improving_performance_by_multiple_steps_within_tffunction) guide instead of [Writing a training loop from scratch](https://www.tensorflow.org/guide/keras/writing_a_training_loop_from_scratch). 

Do let us know if your question about `training_loss.update_state(loss * strategy.num_replicas_in_sync)` is about the code in the TPU guide. Thank you!

**Comment by rohanmuplara** [comment]
@8bitmp3 @rmothukuru sry I had posted wrong link. https://www.tensorflow.org/tutorials/distribute/custom_training

**Comment by rmothukuru** [comment]
> @8bitmp3 @rmothukuru sry I had posted wrong link. https://www.tensorflow.org/tutorials/distribute/custom_training

@rohanmuplara,
Code is not present even in the link you specified,  https://www.tensorflow.org/tutorials/distribute/custom_training. Am I missing something?

**Comment by rohanmuplara** [comment]
Hey sorry @rmothukuru I was completely wrong again and @8bitmp3 was right. I had another question on that link. This is the right link https://www.tensorflow.org/guide/tpu#distribution_strategies.  I have also attached a screen recording of my question.  https://share.descript.com/view/lh1VoY2ojQI

**Comment by rohanmuplara** [comment]
@rmothukuru following up here

**Comment by 8bitmp3** [comment]
Also feel free to ask this or any other question on the TensorFlow Forum - https://discuss.tensorflow.org/ - we have members of the TF community, core TF, and TF DevRel there 👍
